README.adoc

```diff
@@ -2,12 +2,12 @@
 
 = Spring Framework on Google Cloud Platform
 
-This project makes it easy for Spring users to run their applications on Google Cloud Platform.
-You can check our project website https://cloud.spring.io/spring-cloud-gcp[here].
+The Spring Cloud GCP project makes it easy for Spring users to run their applications on Google Cloud Platform.
+You can check our https://github.com/spring-cloud/spring-cloud-gcp/blob/master/README.adoc[project website] for more information.
 
-For a deep dive into the project, refer to the https://cloud.spring.io/spring-cloud-static/spring-cloud-gcp/1.2.0.RELEASE/reference/html/[Spring Cloud GCP 1.2 Reference Document].
+For a deep dive into the project, refer to the https://cloud.spring.io/spring-cloud-static/spring-cloud-gcp/1.2.0.RELEASE/reference/html/[Spring Cloud GCP 1.2 Reference Document] and https://googleapis.dev/java/spring-cloud-gcp/1.2.2.BUILD-SNAPSHOT/index.html[Javadocs].
```

Comment:
Can we have `1.2.2.RELEASE` version of the docs?

Reply:
I think this may not be possible for 1.2.2; we'll have it for the 1.2.3.RELEASE though. It's just that we can only publish javadocs for branches that has the #2173 included in it.

Reply:
Would it be okay to rely on the snapshot link in the interim?

Reply:
Can you create a temporary branch (branching from the release tag) with the added files?

Reply:
Ahh I see, yes this can be done; will followup shortly and update the PR.

Reply:
Ok, this should be good now.